src/NServiceBus.AcceptanceTesting/Support/ScenarioRunner.cs

```diff
@@ -210,7 +210,7 @@ static void PrintSettings(IEnumerable<KeyValuePair<string, object>> settings)
                             }
                         }
 
-                        await Task.Delay(1).ConfigureAwait(false);
+                        await Task.Yield();
```

Comment:
@timbussmann That's what I was thinking too, but I was trying to come up with a reason why these changes were part of the bug fix...

@danielmarbach Were these changed just for consistency then, or is there something we're missing?